src/Microsoft.DotNet.VersionTools/Automation/GitHubApi/IGitHubClient.cs

```diff
@@ -2,11 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Threading.Tasks;
 
 namespace Microsoft.DotNet.VersionTools.Automation.GitHubApi
 {
-    public interface IGitHubClient
+    public interface IGitHubClient : IDisposable
```

Comment:
Should this be renamed?  `IGitClient` sounds better...  Or `IRemoteGitClient`?

Reply:
I don't think so, at least until I can refactor the models to not be specific to GitHub API responses. Ideally what makes the most sense to me is actually layering over both of them to avoid the gaps between how GitHub and VSTS model things.